packages/react-server/package.json

```diff
@@ -24,50 +24,50 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "cls-q": "1.1.0",
-    "cookie": "0.1.2",
+    "cookie": "^0.3.1",
     "cookie-parser": "1.4.3",
-    "eventemitter3": "0.1.6",
-    "express-state": "1.2.0",
+    "eventemitter3": "^2.0.2",
+    "express-state": "^1.4.0",
     "lodash": "^4.16.4",
-    "mobile-detect": "1.3.0",
+    "mobile-detect": "^1.3.5",
     "q": "1.4.1",
     "querystring": "0.2.0",
-    "request-local-storage": "1.1.0",
-    "routr": "0.1.3",
-    "winston": "2.2.0"
+    "request-local-storage": "^1.1.1",
+    "routr": "^2.1.0",
```

Comment:
Let me rephrase this.  The breaking change in Routr *may or may not* impact us.  It all has to do with what we use/expect regarding the `type` property on [this line](https://github.com/redfin/react-server/blob/master/packages/react-server/core/context/Navigator.js#L53).  Our tests might not catch this for two reasons: we don't have test coverage of this case or the change in Routr actually doesn't impact us.

Secondly, the bug identified in #312 means that we're not even passing the HTTP method to Routr for it to check.  The fix for their API change and #312 is to pass the HTTP method to Routr.  *That* change is a breaking change for *our* project because now people will have to allow POST requests in their `routesFile` or else Routr won't select them.